.eslintrc.js

```diff
@@ -1,9 +1,64 @@
 module.exports = {
-  root: true,
-  extends: '@react-native-community',
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
+  parserOptions: {
+    project: `./tsconfig.json`
+  },
+  plugins: ['@typescript-eslint', 'react', 'react-hooks', 'jest'],
+  extends: ['standard-with-typescript', 'plugin:react/recommended', 'plugin:react-hooks/recommended', 'prettier'],
   env: {
-    'jest/globals': true,
+    es6: true,
+    'jest/globals': true
+  },
+  ignorePatterns: ['**/reports/', '**/node_modules/', '**/ios/main.jsbundle', '**/dist/', '**/lib-dist/'],
+  rules: {
+    'arrow-parens': ['error', 'as-needed'],
+    curly: ['error', 'all'],
+    'no-loop-func': 'error',
+    'prefer-const': 'error',
+    'prefer-object-spread': 'error',
+    'prefer-template': 'error',
+
+    'react/display-name': 'off',
+    'react/no-access-state-in-setstate': 'error',
+    'react/no-did-mount-set-state': 'warn',
+    'react/no-did-update-set-state': 'warn',
+    'react/no-unescaped-entities': 'off',
+    'react/no-unused-prop-types': 'warn',
+    'react/no-typos': 'error',
+    'react/prefer-es6-class': ['error', 'always'],
+
+    'react-hooks/rules-of-hooks': 'error',
+    'react-hooks/exhaustive-deps': 'error',
+
+    // TODO Use errors again for these typescript rules
+    '@typescript-eslint/explicit-function-return-type': 'warn',
+    '@typescript-eslint/no-floating-promises': 'warn',
+    '@typescript-eslint/restrict-template-expressions': 'warn',
+    '@typescript-eslint/strict-boolean-expressions': 'warn',
```

Comment:
I had to disable these because there where quite some errors :/

Reply:
but we should fix tem in the future, shouldn't we? Can you open a ticket for it?

Reply:
Done, https://issues.tuerantuer.org/browse/LUN-55